toolkit/tools/internal/network/network.go

```diff
@@ -41,9 +41,8 @@ func DownloadFile(url, dst string, caCerts *x509.CertPool, tlsCerts []tls.Certif
 		RootCAs:      caCerts,
 		Certificates: tlsCerts,
 	}
-	transport := &http.Transport{
-		TLSClientConfig: tlsConfig,
-	}
+	transport := http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
I think this is a welcome change as it sets sane value to most esp `TimeOut`: 
```go
  var DefaultTransport RoundTripper = &Transport{
          Proxy: ProxyFromEnvironment,         
          DialContext: defaultTransportDialContext(&net.Dialer{
                  Timeout:   30 * time.Second,
                  KeepAlive: 30 * time.Second,         
          }),
          ForceAttemptHTTP2:     true,
          MaxIdleConns:          100,
          IdleConnTimeout:       90 * time.Second,
          TLSHandshakeTimeout:   10 * time.Second,
          ExpectContinueTimeout: 1 * time.Second,                      
  }
```

Reply:
Seems applicable to 2.x as well.  I also searched for other uses of the HTTP client and it looks like it all comes through this file.

Now I'm working through `tdnf` proxy fun but that's solely chroot env settings :-)